packages/client/src/library/Svg/Confirm.js

```diff
@@ -0,0 +1,19 @@
+const Confirm = ({ width = "17", height = "16", className = "" }) => (
```

Comment:
Can we rename this svg to describe the icon (for example, CircleCheck)? It's easier to be referenced by other components. Same with Execute

Reply:
Not sure which other components would use them, it looks like it would be used only for confirming/executing, plus these icons might be changed so I would leave the name as-is for now, and change it when we get the final design.

Reply:
Updated the names to match the look of the icon!